commands/cat_end.py

```diff
@@ -0,0 +1,49 @@
+from db.dao.sector_dao import SectorDao
+from base.command import BaseCommand
+from base.arguments import StringArgument
+from db.base import sector_size
+from db.dao.file_dao import FileDao
+
+
+class CatEndCommand(BaseCommand):
+    command = 'cat_end'
+    arguments = [StringArgument()]
+
+    def run(self):
+        filename = self.arguments[0].data
+        current_directory = self.context.current_directory
+
+        if (FileDao.is_unique_filename(filename, current_directory)):
+            self.log('InvalidEntry: File does not exist', prefix=False)
+            # raise ValueError('File does not exist')
+        else:
+            file = FileDao.get_file(filename, current_directory)
+            text = self.get_input('Start Writing: ', prefix=False)
```

Comment:
Not for this PR, but we might want to add a method that accepts multiline input.

Reply:
do what...

Reply:
Right now, whatever the user types, it is limited to a single line. 
Eventually we will want to have something so that user can type in multiple lines.

An example would be `nano file.txt` command. It's a full blown text editor, we just need that multiline editing capability.